app/assets/stylesheets/_mixins.scss

```diff
@@ -44,8 +44,8 @@ $default-border-radius: 3px;
     transition : $type $speed $easing;
 }
 
-@mixin animate($name, $speed:$speed, $occurances:"") {
-    animation : $name $speed $occurances;
+@mixin animate($name, $speed:$speed, $occurrences:"") {
+    animation : $name $speed $occurrences;
```

Comment:
- Line should be indented 2 spaces, but was indented 4 spaces
- Property name should be immediately followed by a colon